views/BeerView.js

```diff
@@ -0,0 +1,61 @@
+var BeerView = Backbone.View.extend({
+  className: 'beer',
+  
+  events: {
+    'click .remove-beer': 'removeBeer',
+    'dblclick .view': 'editBeer',
+    // 'blur .edit': 'saveBeer',
+    'keypress .editName': 'updateOnEnter'
+  },
+
+  template: Handlebars.compile($('#beer-template').html()),
+
+  initialize: function () {
+    this.listenTo(this.model, 'destroy', this.remove),
+    this.listenTo(this.model, 'change', this.render)
+  },
+
+  render: function() {
+    this.$input = this.$('.edit');
+    this.$nameInput = this.$('.editName');
+    this.$el.html(this.template(this.model.toJSON()));
+
+    return this;
+  },
+
+  removeBeer: function () {
+    this.model.destroy();
+  },
+
+  editBeer: function() {
+    $('.view').addClass('hidden');
+    $('.edit').addClass('editing');
+    this.$input.focus();
+  },
+
+  updateOnEnter: function(e) { // Updates the item when we hit enter
+    if (e.which === 13) { // && this.$nameInput.val() just to verify it's not empty
+      this.model.set({name: this.$('.editName').val()});
```

Comment:
I'm not sure you can use set this way:

`this.model.set('name', this.$('.editName').val());`